css/style.css

```
@@ -162,3 +161,50 @@ li {
 .border {
   border-bottom: 1px solid var(--neutrals-n-40);
 }
+
+footer {
+  background-color: var(--neutrals-n-0);
+}
+
+.contact {
+  background: url(../images/footer-bg.svg) var(--primary-b-400) top right no-repeat;
+  border-top-left-radius: 10rem;
+}
+
+.contact__heading {
+  color: var(--neutrals-n-0);
+  text-align: center;
+  font: 700 2.5rem/3.25rem Poppins, sans-serif;
+}
+
+.contact__info {
+  color: var(--primary-b-50);
+  text-align: center;
+  font: 400 1.25rem/1.75rem Poppins, sans-serif;
+}
+
+.input {
+  background: var(--neutrals-n-0);
+  border: 1px solid var(--neutrals-n-40);
+  border-radius: 0.5rem;
+}
+
+input:focus {
+  outline-color: var(--primary-b-300);
+}
+
+textarea:focus {
+  outline-color: var(--primary-b-300);
+}
```

- [ ] Kindly avoid code repetition in order to obey DRY principle. 👍 
```suggestion
input:focus,
textarea:focus {
  outline-color: var(--primary-b-300);
}
```